src/orange/OrangeParams.cc

```diff
@@ -97,11 +114,10 @@ OrangeParams::OrangeParams(const std::string& json_filename)
  * Volume and surface labels must be unique for the time being.
  */
 OrangeParams::OrangeParams(OrangeInput input)
+    : unit_indexer_(make_unit_indexer(input.units))
 {
-    CELER_VALIDATE(input.units.size() == 1,
-                   << "input geometry has " << input.units.size()
-                   << "universes; at present there must be a single global "
-                      "universe");
+    CELER_VALIDATE(!input.units.empty(),
+                   << "input geometry has must contain at least one universe");
```

Comment:
Minor comment, but I've tried to be very consistent about how we write error messages (since they're always presented as `error: <blah>`): see https://github.com/celeritas-project/celeritas/blob/95fb1e9c8e72acba7800f053ccbd4887a64622e0/src/corecel/Assert.hh#L52 .

```suggestion
                   << "input geometry has no universes");
```
(only after editing this did I notice `has must` 😄 )